common/service/dynamicconfig/config.go

```diff
@@ -289,6 +292,18 @@ func (c *Collection) GetBoolPropertyFnWithDomainFilter(key Key, defaultValue boo
 	}
 }
 
+// GetBoolPropertyFnWithDomainIDFilter gets property with domainID filter and asserts that it's a bool
+func (c *Collection) GetBoolPropertyFnWithDomainIDFilter(key Key, defaultValue bool) BoolPropertyFnWithDomainIDFilter {
```

Comment:
Will create similar functions for other data types if decided to merge into master.